### ISSUE TITLE
Batocera Support

### ISSUE THREAD

**Issue by vaderag** [ruckage/es-theme-snes-mini#25]
Just wanted to say that I _love_ this theme - have now moved to Batocera so would love to see it supported!
Thanks!

**Comment by thias** [comment]
See https://github.com/thias/es-theme-snes-mini and https://github.com/ruckage/es-theme-snes-mini/issues/22#issuecomment-1105827517 which is enough for me to have the theme usable with Batocera, with quick and easy changing of all config changes straight from the UI settings menu.

**Comment by vaderag** [comment]
Thanks, so the change of 

`<pos>${listx} 0.216</pos>
<size>${listWidth} 0.578</size>
<lineSpacing>1.375</lineSpacing>
<selectorHeight>0.078</selectorHeight>`

to **/layouts/1920x1080.xml**  definitely helped, however it's not all solving

1. Default layout... while it's possible I changed my layout but I'm pretty certain it's default as it matches on other themes, B is Launch and X certainly isn't random in Batocera.  Basically the button layout descriptions don't match
2. Favorite, achievements, manual icons do not scale
![image](https://user-images.githubusercontent.com/17175988/164680212-c9a63625-00a5-4405-bae5-3e7d7c594622.png)

4. The clock is mispositioned
![image](https://user-images.githubusercontent.com/17175988/164680119-0311e077-63a5-451b-bc3f-8ae80203123d.png)

Most of these are probably solveable with a dive into the code - i'll take a look if i get chance

**Comment by thias** [comment]
Yeah, same here... the star on the left of the game name for favorites is also too high. And for some systems I also see some lists with position issues, but not all of them, and not always the same... super weird.
There's a quick "fix" for the clock: Disabling it from the options :sweat_smile:
If you do look into fixing this properly for Batocera's EmulationStation, don't hesitate to create pull requests in my fork, as I'm also interested.

**Comment by bulzipke** [comment]
> Yeah, same here... the star on the left of the game name for favorites is also too high. And for some systems I also see some lists with position issues, but not all of them, and not always the same... super weird. There's a quick "fix" for the clock: Disabling it from the options 😅 If you do look into fixing this properly for Batocera's EmulationStation, don't hesitate to create pull requests in my fork, as I'm also interested.

Hi,

I know this is an old issue, but I have a question. I absolutely love this project and, hoping it would continue to thrive on Batocera as well, I forked it and made some modifications. I plan to keep working on it in the future too. However, I recently noticed that you also forked this project.

If your fork is something you intend to maintain moving forward, I was wondering if it might be better to manage things through your fork instead. What do you think? I don’t mean to burden you, and I sincerely apologize if I came across as rude.

This is my fork.
https://github.com/bulzipke/es-theme-snes-mini

**Comment by thias** [comment]
No, I don't expect to be maintaining my fork. Feel free to pick whatever changes that might still be relevant. I'll definitely check your fork when I revisit my Batocera SNES Pi!

**Comment by bulzipke** [comment]
> No, I don't expect to be maintaining my fork. Feel free to pick whatever changes that might still be relevant. I'll definitely check your fork when I revisit my Batocera SNES Pi!

Thank you for your response. I hope this theme will continue to be maintained!

**Comment by ViperAcidZX** [comment]
> Thank you for your response. I hope this theme will continue to be maintained!

Can you please also add support for Recalbox too? I was going to use Batocera on my Pi 3B+ but it won't cooperate with the safe shutdown and reset scripts on my SuperPi case and attempts of using forks I once used only gave me problems and corrupted my install.
